resources/zib/zib-BodyTemperature.xml

```diff
@@ -0,0 +1,151 @@
+      <alias value="Lichaamstemperatuur" />
+      <mapping>
+        <identity value="zib-bodytemperature-v3.1.2-2020EN" />
+        <map value="NL-CM:12.6.1" />
+        <comment value="BodyTemperature" />
+      </mapping>
+    </element>
+    <element id="Observation.code.coding">
+      <path value="Observation.code.coding" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.code.coding:BodyTempCode">
+      <path value="Observation.code.coding" />
+      <sliceName value="BodyTempCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCoding>
+        <system value="http://loinc.org" />
+        <code value="8310-5" />
+      </patternCoding>
+    </element>

```

Comment:
I think you mean to map this on `observation.code`

Reply:
Observation.code has a binding to both the definitioncode and to TemperatureTypeCodelist, so it needs to be sliced.